src/test/java/AddUserTest.java

```diff
@@ -0,0 +1,98 @@
+import client.UserClient;
+import client.ZipCodeClient;
+import com.github.javafaker.Faker;
+import data.Response;
+import data.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+import static data.Constants.*;
+
+public class AddUserTest {
+    private static UserClient userClient;
+    private static ZipCodeClient zipCodeClient;
+    private static Faker faker;
+
+    @BeforeEach
+    public void setup() {
+        userClient = new UserClient();
+        zipCodeClient = new ZipCodeClient();
+        faker = new Faker();
+    }
+
+    // Scenario #1:
+    @Test
+    void addUserTest() throws IOException {
+        String zipCode = zipCodeClient.getNewZipCode();
+        User user = User.builder()
+                .name(faker.name().firstName())
+                .age(faker.number().numberBetween(1, 100))
+                .sex(faker.demographic().sex().toUpperCase())
+                .zipCode(zipCode)
+                .build();
+        int postUserStatusCode = userClient.addToUsersList(user);
+        Response<List<User>> getUsersResponse = userClient.getUsersList();
```

Comment:
you named your local variables like methods starting with verb. This is not good